sql/utils/ssh_tunnel.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding:utf-8 -*-
+"""
+@author: EliasChiang
+@license: Apache Licence
+@file: ssh_tunnel.py
+@time: 2020/05/09
+"""
+from sshtunnel import SSHTunnelForwarder
+from paramiko import RSAKey
+
+class SSHConnection(object):
+    """
+    ssh隧道连接类，用于映射ssh隧道端口到本地，连接结束时需要清理
+    """
+    def __init__(self, host, port, tun_host, tun_port, tun_user, tun_password, pkey_address, pkey_password):
+        self.host = host
+        self.port = int(port)
+        self.tun_host = tun_host
+        self.tun_port = int(tun_port)
+        self.tun_user = tun_user
+        self.tun_password = tun_password
+
+        if pkey_address != None and pkey_address != "":
+            self.private_key = RSAKey.from_private_key_file(pkey_address, password=pkey_password)
```

Comment:
判断非空也可以直接if, 另外建议 pkey_address 改成 pkey_path
```suggestion
        if pkey_address and pkey_path:
            self.private_key = RSAKey.from_private_key_file(pkey_path, password=pkey_password)
```

Reply:
要改的话都字段什么的都要改一下

Reply:
已经修改了非空判断，和pkey_address 改成 pkey_path